sql/wsrep_mysqld.cc

```diff
@@ -3003,6 +3009,15 @@ enum wsrep::streaming_context::fragment_unit wsrep_fragment_unit(ulong unit)
   }
 }
 
+bool THD::wsrep_wait_for_parallel_slave_prior_commit() 
+{
```

Comment:
the name of the method is a bit misleading, as it is the parallel slave who is waiting. I suggest to change to:
wsrep_parallel_slave_wait_for prior_commit()